lib/pxenforcer.js

```diff
@@ -55,6 +55,14 @@ class PxEnforcer {
             return cb();
         }
 
+        try {
+            if(telemetryHandler.isTelemetryCommand(req, this._config)) {
+                this.pxClient.sendEnforcerTelemetry('command', this._config);
+            }
+        } catch (err) {
+            this.logger.error('Telemetry command failure. unexpected error. ' + err.message);
```

Comment:
make this a debug message. it shouldn't bother the customer that this request failed, and they don't need to see it in their error logs

Reply:
changed